apps/ewallet/priv/repo/report_minimum.exs

```diff
@@ -1,9 +1,10 @@
 alias EWallet.CLI
+alias EWalletConfig.Config
 alias EWalletDB.AuthToken
 
 # :prod environment does not have a default :base_url value and should not have one.
 # But we have a fallback value here so we can generate a friendly output message for seeding.
-base_url = Application.get_env(:ewallet_db, :base_url) || "https://example.com"
+base_url = Application.get_env(:ewallet, :base_url) || "https://example.com"
```

Comment:
Do we still need the default `https://example.com/`? 🤔

Reply:
I guess not, removing it!